data/strings/types_strings.txt

```diff
@@ -12088,6 +12089,7 @@
     et = Sissesõidutee
     fi = Ajotie
     mr = चालकमार्ग
+    pl = Droga podjazdowa
```

Comment:
```suggestion
    pl = Podjazd
```

Reply:
Note that you can apply this change if considered as improvement

Reply:
Your suggestion is OK for me ;-)